Release docs 2.9 (#177)
openmsupply/msupply_docs: content/docs/dispensary/prescriptions/index.md

3. A new window opens, in which you can select a patient and other details for the prescription

![New Prescription Window](images/prescription_new_prescription_name.png)

#### Select a Patient

In the `New Prescription` window, you will be presented with a form to search for available patients. The patients list will load when you start typing - you can enter as much of the patient's name or code as you need.

> If no match is found a `New patient` button will show. Clicking this will open a section to enter the patient's details.

Reviewer: ```suggestion
If no match is found, a `New patient` button will appear. Click this to open a section where you can enter the patient's details.
```

![Create new patient pop up](images/prescription_new_prescription_no_match.png)

Enter the patient's first name and last name to enable the `Search` button. This will show any matching patients even if they were created on a different mSupply site.

![Create new patient window](images/prescription_new_prescription_create_patient.png)

Click on a patient here to select them for for the prescription, or select `Create new patient` to continue to the next stage and add the remaining details for the new patient. Select `Save` to save the patient and continue creating the new prescription.

<div class="note">

> If you have found a matching patient, but it's not the correct one, you can just press space a few times to see the New Patient button.</div>

Reviewer: What do you mean press space a few times...? Do you not just press the `Create new patient` button?

Author: This is because the `Create Patient` button only shows if there aren't any search results. So if the patients name has search results, but that result is actually a different patient with the same name, the search needs to be forced to return no results before creating is an option

Reviewer: Ah right... we so have to improve this 👀 can't get the new creation button to show every time there's a search result? Not change request btw, more like a OMS issue creation request XD

Author: That's easy to do! The default behaviour for the new autocomplete button is to always show - I'd added extra logic in Patients to only show when there's no results

> #### Select a Clinician

Reviewer: ```suggestion
#### Clinician
```

> Optional field for the clinician who prescribed the medicine. You can search for an existing clinician by code or name. If the clinician is not in the search results you can click `Create Clinician` to enter a new clinician.

Reviewer: This button should be sentence case in the app... will need oms changes 👀 

```suggestion
This is an optional field for the clinician who prescribed the medicine. You can search for an existing clinician by code or name. If no match is found, click `Create clinician` to add a new one.
```

![New clinician](images/clinicians_search_new_prescription.png)

> Complete the details for the clinician. Clicking `Create` will save the clinician and add them to the new prescription.

Reviewer: ```suggestion
Enter the clinician’s details, then click `Create` to save the clinician and add them to the prescription.
```

![Prescription: created](images/prescription_created.png)

> ### Changing the Patient

Reviewer: I don't think our docs are grouped by flow =/ but if it's by flow then no... If you look at Inbounds you can see it's Select customer -> edit customer

> If you have selected the wrong patient, you can change the patient name in the `Patient` field by typing some characters to search and then selecting from the dropdown list. You can also create a new patient here if no match is found.

Reviewer: ```suggestion
If you’ve selected the wrong patient, you can change the patient by typing in the `Patient` field to search and select the correct one from the dropdown list. If no match is found, you can also create a new patient from here.
```

You may need to type more of the patient's name or code if you have more than 100 patients for the patient's name to show on the dropdown list.

> ### Edit the Patient Details

Reviewer: Same here group with patient infomation pls

> You can add or edit the details of the selected patient by clicking on the pencil icon next to the patients name. Click `Save` to save any changes or `View Patient` to go to the patients file.

Reviewer: ```suggestion
To edit the details of the selected patient, click the pencil icon next to the patient’s name. After making changes, click Save to update the information, or select `View Patient` to open the patient’s file. 
```

![Edit patient](images/prescription_edit_patient_step_1.png)
![Edit patient](images/prescription_edit_patient_step_2.png)

> Insurance details can be viewed in the Insurance tab if you have setup <a href="https://docs.msupply.org.nz/dispensing:patient_insurance#insurance_providers">Insurance Providers in mSupply</a>

Reviewer: Another link docs/dispensary/patients/#insurance

> ### Changing the Clinician

Reviewer: Would move up to clinician to keep together

> If you have selected the wrong clinician, you can change the clinician name in the `Clinician` field by typing some characters to search and then selecting from the dropdown list. You can also create a new clinician here if no match is found.

Reviewer: ```suggestion
If you’ve selected the wrong clinician, you can change the clinician by typing in the `Clinician` field to search and select the correct one from the dropdown list. If no match is found, you can also create a new clinician from here.
```
